lib/tendermint-light-client-types/src/client_state.rs

```diff
@@ -95,6 +102,8 @@ pub mod proto {
                     .collect::<Result<Vec<_>, _>>()
                     .map_err(Error::ProofSpecs)?,
                 upgrade_path: value.upgrade_path,
+                // // FIXME: we need to define the tm proto ourself and regenerate
```

Comment:
change comment to instead mention that the contract address field is not used for the ibc-go native implementation